rdr_service/offline/main.py

```diff
@@ -938,6 +945,12 @@ def _build_pipeline_app():
         view_func=genomic_gem_a3_workflow,
         methods=["GET"]
     )
+    offline_app.add_url_rule(
+        OFFLINE_PREFIX + "GenomicUpdateReportStateForConsentRemoval",
+        endpoint="update_report_state_for_consent_removal",
+        view_func=update_report_state_for_consent_removal,
+        methods=["GET"]
+    )
```

Comment:
Should the cron job be scheduled (like in cron_default.yaml), or is it intentional to leave it unscheduled for now?

Reply:
Yes it should be scheduled. I completely forgot to add it to the cron schedule, thanks!